test/unit/adversarial/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
```

Comment:
why do we need this file?

Reply:
It is also on all other subdirs of /test/unit, but i don't know